docs/csharp/language-reference/configure-language-version.md

```diff
@@ -20,6 +20,17 @@ You can override the default behavior by specifying a language version. There ar
 - Set the language version [for multiple projects in a subdirectory](#configure-multiple-projects).
 - Configure the [`-langversion` compiler option](#set-the-langversion-compiler-option).
 
+## Required SDK and Runtime version
```

Comment:
Now there is a disconnect between this heading and the following text. If the text talks about target frameworks, but not about SDKs or runtimes, I think the heading should too.

Reply:
Thank you. I was looking so closely at the previous comments that I didn't see this.